changelog/V-VAtsIKQuGv2Q-_LVDz1w.md

```diff
@@ -0,0 +1,6 @@
+audience: general
+level: minor
```

Comment:
Maybe patch, since no new functionality is added?

Reply:
sure, yes